lms/services/lti.py

```diff
@@ -0,0 +1,86 @@
+"""
+LTI service.
+
+This is a place to put LTI-related methods where they can be called from
+anywhere in the code, they can have access to the request and context, and they
+can have state (e.g. to cache the results of computations).
+"""
+import pylti.common
+
+from lms.services import LTILaunchVerificationError
+from lms.services import NoConsumerKey
+from lms.services import ConsumerKeyError
+from lms.services import LTIOAuthError
+
+
+class LTIService:
+    """A collection of LTI-related methods."""
+
+    def __init__(self, _context, request):
+        self._request = request
+        self._called = False
```

Comment:
Since this service's description says it may have other methods in future, `_called` is perhaps an ambiguous name.

Reply:
Changed to `_request_verified`